src/js/common/components/form-elements/ErrorableCurrentOrPastDate.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import moment from 'moment';
+
+import ErrorableDate from './ErrorableDate';
+import { isDirtyDate } from '../../utils/validations';
+import { dateToMoment } from '../../utils/helpers';
+
+export default function ErrorableCurrentOrPastDate(props) {
```

Comment:
How does this access the props? I'm assuming these are the props of `ErrorableDate`?